pkg/minikube/console/style.go

```diff
@@ -130,29 +131,46 @@ func lowPrefix(s style) string {
 	return lowBullet
 }
 
-// Apply styling to a format string
-func applyStyle(style StyleEnum, useColor bool, format string, a ...interface{}) string {
-	p := message.NewPrinter(translate.GetPreferredLanguage())
-	for i, x := range a {
-		if _, ok := x.(int); ok {
-			a[i] = number.Decimal(x, number.NoSeparator())
-		}
-	}
+// applyStyle translates the given string if necessary then adds any appropriate style prefix.
+func applyStyle(style StyleEnum, useColor bool, format string) string {
 	format = translate.T(format)
-	out := p.Sprintf(format, a...)
 
 	s, ok := styles[style]
 	if !s.OmitNewline {
-		out += "\n"
+		format += "\n"
 	}
 
 	// Similar to CSS styles, if no style matches, output an unformatted string.
 	if !ok {
-		return p.Sprintf(format, a...)
+		return format
 	}
 
 	if !useColor {
-		return applyPrefix(lowPrefix(s), out)
+		return applyPrefix(lowPrefix(s), format)
+	}
+	return applyPrefix(s.Prefix, format)
+}
+
+func applyTemplateFormatting(style StyleEnum, useColor bool, format string, b map[string]interface{}) string {
```

Comment:
just curious - what does `b` stand for?

Reply:
It was `b` when I was testing OutStyle being one function for both versions. Changed back to `a`.